documentation/modules/ROOT/pages/connectors/sqlserver.adoc

```diff
@@ -1737,17 +1672,176 @@ Typically, you configure the {prodname} SQL Server connector in a `.json` file u
 endif::community[]
 
 ifdef::product[]
+To deploy a {prodname} SQL Server connector, you add the connector files to Kafka Connect, create a custom container to run the connector, and then add connector configuration to your container.
+For details about deploying the {prodname} SQL Server connector, see the following topics:
+
+* xref:deploying-debezium-sql-server-connectors[]
+* xref:sql-server-connector-properties[]
+endif::product[]
+
+ifdef::product[]
+// Type: procedure
+// ModuleID: deploying-debezium-sql-server-connectors
+[[sql-server-deploying-a-connector]]
+=== Deploying {prodname} SQL Server connectors
+
+To deploy a {prodname} SQL Server connector, you must build a custom Kafka Connect container image that contains the {prodname} connector archive, and then push this container image to a container registry. 
+You then need to create the following custom resources (CRs):
+
+* A `KafkaConnect` CR that configures your Kafka Connector and that specifies the name of the image that you created to run your {prodname} connector.
+  You apply this CR to the OpenShift Kafka instance.
+
+* A `KafkaConnector` CR that configures your {prodname} SQL Server connector.
+  You apply this CR to the OpenShift instance where Red Hat AMQ Streams is deployed.
+
+
+.Prerequisites
+
+* SQL Server is running and you completed the steps to {LinkDebeziumUserGuide}#setting-up-sql-server-for-use-with-the-debezium-sql-server-connector[set up SQL Server to run a {prodname} connector].
+
+* link:https://access.redhat.com/products/red-hat-amq#streams[Red Hat AMQ Streams] was used to set up and start running Apache Kafka and Kafka Connect on OpenShift.
```

Comment:
Shouldn't the first occurrence of "Red Hat AMQ Streams" be linked? I see it's mentioned above (line 1695) and potentially earlier, too?

Reply:
Yes, good point. The content that is presented here mirrors what's in the current PG doc, so the same comment applies there and elsewhere.  I actually found the link on L1702 to be a little misleading, because I expected it to target instructions that describe how to perform the Kafka and Kafka Connect setup, and not just take me to a product page.

Reply:
Yes, sounds good.